README.rst

```diff
@@ -20,251 +27,22 @@ of critical security issues) or with an extensive deprecation period.
 Issues and requests against GA libraries are addressed with the highest
 priority.
 
+GA libraries have development status classifier ``Development Status :: 5 - Production/Stable``.
+
 .. note::
 
     Sub-components of GA libraries explicitly marked as beta in the
     import path (e.g. ``google.cloud.language_v1beta2``) should be considered
     to be beta.
 
-The following client libraries have **GA** support:
```

Comment:
I can appreciate the intent here, but I rather like what we have for nodejs and java:
- https://github.com/googleapis/google-cloud-node
- https://github.com/googleapis/google-cloud-java

Both are automated, and should be easy to use as an example.  Can we keep this?